articles/quickstarts/search.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Q# quickstart: TODO"
+description: TODO
+author: cgranade
+ms.author: chgranad@microsoft.com
+ms.date: 10/9/2019
+ms.topic: article
+uid: microsoft.quantum.quickstarts.search
+---
+
+# Search for marked inputs with Grover's Algorithm
+
+This introduction uses [Grover's Algorithm][grover] to search inputs to a function for a particular _marked input_.
+This algorithm can be used to search for inputs with some special property, to find items in an unstructured list, and so forth.
+Grover's algorithm lets us use a quantum computer to run this search in less steps than the number of items in the list that we're searching — something no classical algorithm can do.
+
+In this introduction, we see how we can use Q# and the Quantum Development Kit to run Grover's algorithm with just a few lines of code.
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit][install].
+
+## Writing Grover's Algorithm
+
+Using the Quantum Development Kit, create a new Q# project:
+
+```
+$ dotnet new console -lang Q# --output Grover
+$ cd Grover
+```
+
+Grover's algorithm itself can be written using just a few lines.
+Add the following code to the `Operations.qs` file in your new project:
+
+[!code-qsharp[](~/quantum/samples/algorithms/simple-grover/SimpleGrover.qs?highlight=5,27)]
```

Comment:
Generally for this kind of tutorial, does additional information live in code comments or as docs text with a snippet underneath? In the TensorFlow example the code is more broken up into steps rather than one large block to copy paste. Having the big block is easy for copy paste but doesn't really take advantage of it being a walk through (because you can copy it and run it without reading any of the comments). @natke , is there a docs standard for how to do this?

Reply:
I hope that this sample can get a bit shorter as we bring more of the contents of `Reflections.qs` into the libraries themselves as convenience operations.

Reply:
I definitely agree with breaking the code up in cases like the Bell state preparation sample, but my hope is that we can move a bunch of operations over to the M.Q.Preparation and M.Q.AmplitudeAmplification namespaces (e.g.: ReflectAboutUniform isn't really specific to this sample). As we do so, this article should automatically update thanks to the source code links that @natke helped to get working.